add-chain/testdata/monorepo/deployments/.deploy

```diff
@@ -0,0 +1,11 @@
+{
+	"AddressManager": "0xBBCfC68f590ae60Feb1551F4C639E3Eab5ea7BA9",
+	"L1CrossDomainMessengerProxy": "0x6b95418119Aa1bc41D521377D98E6c367817A755",
+	"L1ERC721BridgeProxy": "0x460Cb843a61CeB6cFB3BC3373FC405744f5be4ee",
+	"L1StandardBridgeProxy": "0xC4Ed1Ae4671B38A44B3cea27E1F7462cFe6B1A87",
+	"L2OutputOracleProxy": "0x3F8A862E63E759a77DA22d384027D21BF096bA9E",
```

Comment:
Can we have a second test for chains on fault proofs? These would have no L2OO, but would have other contracts like the Dispute Game Factory. OP Sepolia can be used as a reference here for now

Reply:
> Can definitely add this test. Is [this](https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/extra/addresses/sepolia/op.json) the file you want to use as a reference? Note that it does include a `L2OutputOracleProxy` address.

Good call. I opened https://github.com/ethereum-optimism/superchain-registry/pull/218 to fix this, but it's actually failing, so we'll need a fix on the checks

> Would you expect the `add-chain` script to behave differently for chains on fault proofs other than the output file in `superchain/extra/addresses/<chain_name>.json` would have a different set of contracts?

I don't think so, but the checks required are different (e.g. we have no L2OO params to check, and we'll need to define standard config params for fault proofs)

Reply:
Issue created here to capture the work involved with this update: https://github.com/ethereum-optimism/superchain-registry/issues/222. Will address in a follow-up PR